webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -199,11 +199,11 @@ public function indexAction(Request $request): Response
             'filteredResults' => $filteredVerdicts,
             'showExternalResult' => $this->config->get('data_source') ==
                 DOMJudgeService::DATA_SOURCE_CONFIGURATION_AND_LIVE_EXTERNAL,
+            'showTestcases' => count($submissions) <= 50,
```

Comment:
I think this magic 50 is related to the same 50 used on line 139. Please store this cutoff number in one place only.

Reply:
Done